themes/classic/templates/_partials/form-fields.tpl

```diff
@@ -136,10 +136,10 @@
             <input
               class="form-control js-child-focus js-visible-password"
               name="{$field.name}"
-              title="{l s='At least 5 characters long' d='Shop.Forms.Help'}"
+              title="{l s='At least 8 characters long, 1 lowercase letter, 1 uppercase letter, 1 digit and 1 special char' d='Shop.Forms.Help'}"
               type="password"
               value=""
-              pattern=".{literal}{{/literal}5,{literal}}{/literal}"
+              pattern="{literal}((?=.*[A-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[^\w])\S{8,72}|.{5,72}){/literal}"
```

Comment:
Same comment about A-z

Reply:
I inject with `Validate::PASSWORD_LENGTH` this form is too generic to inject a variable with smarty :/